docs/CONTRIBUTING.md

```
@@ -0,0 +1,382 @@
+pnpm lint
+```
+
+### 환경 변수 설정 (.env)
+
+Firebase 연동을 위해 `.env` 파일 설정이 필요합니다.
+
+> `.env` 파일은 절대 GitHub에 커밋하지 마세요. `.gitignore`에 등록되어 있습니다.
+
+환경 변수 값은 **팀 노션**을 참고해서 프로젝트 루트에 `.env` 파일을 생성하세요.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)